tests/test_main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from unittest import mock
+from app.main import cryptocurrency_action
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_predicated_exchange_rate_bigger_then_5(mocked_exchange_rate):
+    mocked_exchange_rate.return_value = 1.49
+
+    assert cryptocurrency_action(1.49) == "Sell all your cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_predicated_exchange_rate_lower_than_5(mocked_exchange_rate):
+    mocked_exchange_rate.return_value = 2
+
+    assert cryptocurrency_action(1.5) == "Buy more cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_predicated_exchange_rate_difference_low(mocked_exchange_rate):
+    mocked_exchange_rate.return_value = 2.1
+
+    assert cryptocurrency_action(2) == "Do nothing"
```

Comment:
Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 0.95 * value passed in `cryptocurrency_action`.